src/pages/Login.vue

```diff
@@ -287,6 +287,7 @@ export default {
             this.invalidOtp = true;
             this.toast.error(this.$t("login.otp.incorrect"));

```

Comment:
1. both submit otp and google sign in button show loading spinner
2. while loading spinner, if someone leaves, pending is still true and so, spinner keeps running